### ISSUE TITLE
Prompt using "modern" theme and selectable git status using something less sus as gitstatus

### ISSUE THREAD

**Issue by marcus-grant** [marcus-grant/dots-bash#2]
The next most important feature after the general aliases is a nice prompt. I liked the [modern prompt theme](https://github.com/Bash-it/bash-it/blob/e06066c36768ac08a31927646a150a6e66badbfc/themes/modern/modern.theme.bash) from Bash-it. It needs to be able to detect whether git status is desired somehow, and deploy [it](https://gist.github.com/wolever/6525437). It should default to **NOT** using git status as it's an extremely expensive computation for a prompt. Map out new features to include that covers displaying in the prompt. Also consider using starship, like on zsh.

**Comment by marcus-grant** [comment]
Here is some info on how starship handles the slowness of git status https://github.com/starship/starship/pull/2465

**Comment by marcus-grant** [comment]
And this is appently an even bigger problem on windows, so consider this approach as well: https://markentier.tech/posts/2020/10/faster-git-under-wsl2/

**Comment by marcus-grant** [comment]
4bcb96789acd3c9dd664dae506f32e0f8f04592a handles ls aliases

**Comment by marcus-grant** [comment]
ebc9e1f6da7cb22ec13bbf3c0dfcf3039fd89be7 handles editing aliases like `edit` for invoking `$EDITOR`, `ebrc` edits `.bashrc`, `ebpf` edits `.bash_profile`, `rbrc` reloads `.bashrc` & `.bash_profile`.

**Comment by marcus-grant** [comment]
a98ca99 renames `.bashrc` to `rc.bash` & `.bash_profile` to `profile.bash`

**Comment by marcus-grant** [comment]
A git status toggler could be set as a part of the general aliases file:
- Set a variable selector for enabling/disabling git status (always start on not having it)
- Have the prompt file conditionally render git status based off it
- Reload bashrc `rbrc` alias

**Comment by marcus-grant** [comment]
There's now a minimal prompt using starship. There's a file, `./prompts/starship-prompt.bash` that checks for starship, installs it if not there, then sets an env var `STARSHIP_CONFIG` to `./prompts/starship-basic.toml` to load up the prompt with `starship init bash`.

Unfortunately the `username` module doesn't seem to work or some hard to find mistake I've made keeps reasserting itself. It should be addressed in issue #5.

Otherwise it has its basic functionality has been implemented, moving prompt selection to the more general issue in #6.

Closing with bed42c8 having resolved the issue
